privacy_policy/en/privacy-notice-product-temp-mail.md

```diff
@@ -0,0 +1,27 @@
+# AdGuard Temp Mail Privacy Notice
+
+At AdGuard Software Limited, we are committed to safeguarding your privacy and ensuring the protection of your data. This Privacy Notice explains the data we collect and process when you use AdGuard Temp Mail, our temporary email address generator service.
+
+## Data Collection and Processing
+
+* **Creation Time of Mailbox:** We retain the timestamp when each mailbox is created.
+* **Last Access Time:** We record the time of the last access to each mailbox.
+* **Mailbox Deletion Time:** The deletion time of mailboxes is stored. Mailboxes can be deleted manually by users or automatically after 7 days of inactivity. The email address is permanently stored and marked as deleted upon removal.
```

Comment:
The reason for permanently storing the email address is to avoid allocating the same address to a different user.